parser/sql/dialect/mysql/src/main/java/org/apache/shardingsphere/sql/parser/mysql/visitor/statement/MySQLStatementVisitor.java

```diff
@@ -916,6 +917,11 @@ public final ASTNode visitAggregationFunction(final AggregationFunctionContext c
                 : new ExpressionProjectionSegment(ctx.getStart().getStartIndex(), ctx.getStop().getStopIndex(), getOriginalText(ctx));
     }
     
+    @Override
+    public ASTNode visitSeparatorName(final MySQLStatementParser.SeparatorNameContext ctx) {
```

Comment:
Please use import static for SeparatorNameContext, and remove `MySQLStatementParser.`.